styles/stepper.css

```diff
@@ -104,3 +104,36 @@
         padding-left: 1rem;
     }
 }
+
+.circle-progress-indicator {
```

Comment:
This shouldn't be a part of the `stepper.css` file and would be more maintainable if it's in its own file.

Reply:
it has its own css now